src/routes/docs/products/auth/teams/+page.markdoc

```diff
@@ -8,7 +8,27 @@ Teams are a good way to allow users to share access to resources.
 For example, in a todo app, a user can [create a team](/docs/references/cloud/client-web/teams#create) for one of their todo lists and [invite another user](/docs/references/cloud/client-web/teams#createMembership) to the team to grant the other user access.
 You can further give special rights to parts of a team using team roles.
 
-The invited user can [accept the invitation](/docs/references/cloud/client-web/teams#updateMembershipStatus) to gain access. If the user's ever removed from the team, they'll lose access again. 
+The invited user can [accept the invitation](/docs/references/cloud/client-web/teams#updateMembershipStatus) to gain access. If the user's ever removed from the team, they'll lose access again.
+
+# Common use cases {% #use-cases %}
+
+## Multi-tenancy
```

Comment:
This part of the docs is for high-level introduction to the concept. I don't think we should delve into a use-case here. This type of content seems to fit more under Journeys. Separating it out as a Journey will also make it more visible.

See https://github.com/appwrite/website/blob/main/STYLE.md#products for info about the different sections and what belongs where.

Reply:
That makes sense. What do you think about the update?